tb_device_mqtt.py

```diff
@@ -843,11 +843,12 @@ def request_attributes(self, client_keys=None, shared_keys=None, callback=None):
         info = self._publish_data(msg, ATTRIBUTES_TOPIC_REQUEST + str(attr_request_number), self.quality_of_service,
                                   high_priority=True)
 
-        self._add_timeout(attr_request_number, ts_in_millis + DEFAULT_TIMEOUT * 1000)
+        self._add_timeout(attr_request_number, ts_in_millis, timeout=20)
         return info
 
-    def _add_timeout(self, attr_request_number, timestamp):
-        self.__timeout_queue.put({"ts": timestamp, "attribute_request_id": attr_request_number})
+    def _add_timeout(self, attr_request_number, timestamp, timeout=DEFAULT_TIMEOUT):
+        timestamp += timeout * 1000
+        self.__attrs_request_timeout[attr_request_number] = timestamp
```

Comment:
It will be better to convert timestamp to int before adding it to the dictionary, because in the final - we will compare with int value.